airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/BigQueryRecordConsumer.java

```diff
@@ -53,7 +56,18 @@ private void processRecord(AirbyteMessage message) {
   @Override
   public void close(final boolean hasFailed) {
     LOGGER.info("Started closing all connections");
-    uploaderMap.values().parallelStream().forEach(uploader -> uploader.close(hasFailed, outputRecordCollector, lastStateMessage));
+    List<Exception> exceptionsThrown = new ArrayList<>();
+    uploaderMap.values().parallelStream().forEach(uploader -> {
```

Comment:
Can we just use normal stream instead of parallel stream?

Reply:
sure